src/aat/resources/features/F-103/F-103_Case_Role_Assign_Base.td.json

```diff
@@ -1,26 +1,29 @@
 {
 	"_guid_": "F-103_Case_Role_Assign_Base",
-	"method": "PUT",
-	"productName": "CCD Data Store",
-	"operationName": "Grant access to a case",
-	"uri": "/cases/{case_id}/users/{user_id}",
-	"users": {
-		"invokingUser": {
-      "_extends_": "BeftaCaseworkerCaa"
-		}
-	},
+
+
+	"productName": "CCD Data Store Api",
+	"operationName": "Add Case-Assigned Users and Roles",
```

Comment:
The operation name "Add Case-Assigned Users and Roles" is for the newly developed one.